docs/documentation/platform/dynamic-secrets/ldap.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "LDAP"
+description: "Learn how to dynamically generate user credentials via LDAP."
+---
+
+The Infisical LDAP dynamic secret allows you to generate user credentials on demand via LDAP. The integration is general to any LDAP implementation but has been tested with OpenLDAP and Active directory as of now.
+
+## Prerequisites
+
+1. Create a user with the necessary permissions to create users in your LDAP server.
+2. Ensure your LDAP server is reachable via Infisical instance.
+
+## Set up Dynamic Secrets with LDAP
+
+<Steps>
+  <Step title="Open Secret Overview Dashboard">
+	Open the Secret Overview dashboard and select the environment in which you would like to add a dynamic secret.
+  </Step>
+  <Step title="Click on the 'Add Dynamic Secret' button">
+	![Add Dynamic Secret Button](../../../images/platform/dynamic-secrets/add-dynamic-secret-button.png)
+  </Step>
+  <Step title="Select 'LDAP'">
+	![Dynamic Secret Modal](../../../images/platform/dynamic-secrets/dynamic-secret-ldap-select.png)
```

Comment:
for images in docs, let's keep a consistent resolution - ideally, full-screen
<img width="814" alt="image" src="https://github.com/user-attachments/assets/cb4c23c4-b97e-47c4-8ef9-31434308d8b0">

please adjust the rest accordingly.
example:
<img width="1253" alt="image" src="https://github.com/user-attachments/assets/c30714da-8817-4254-85ca-24830a9d03f7">